NuGet.config

```diff
@@ -3,7 +3,12 @@
   <packageSources>
     <add key="ASP.NET 5 Dev" value="https://www.myget.org/F/aspnetvnext/" />
     <add key="ASP.NET 5" value="https://www.myget.org/F/aspnetrelease/" />
-    <add key="nuget.org feed" value="https://www.nuget.org/api/v2/" />    
+    <add key="dotnet.myget.org dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/" />
+    <add key="AI myget" value="https://www.myget.org/F/applicationinsights/api/v3/index.json" />
+    <add key="dotnet.myget.org dotnet-cli" value="https://dotnet.myget.org/F/dotnet-cli/" />
+    <add key="coreclr-xunit" value="https://www.myget.org/F/coreclr-xunit/api/v2" />
+    <add key="AspNetCIDev" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
```

Comment:
This nuget.config file has several ASP.NET MyGet feeds on it - that can't be right. You can't mix 'dev' feeds and 'release' feeds. And for that matter, it's mixing different "qualities" of feeds: the "aspnetcidev" feed is untested 'dev' builds (RTM), and the "aspnetrelease" feed is tested and signed 'release' builds for RC2.